sage/pallet/src/mock.rs

```diff
@@ -357,11 +349,45 @@ impl AssetInspector for MockAssetMediator {
 	}
 }
 
-pub type SageInstance1 = pallet_sage::Instance1;
-impl crate::Config<SageInstance1> for Test {
+#[cfg(feature = "runtime-benchmarks")]
+pub struct SageBenchmarkHelper;
+
+#[cfg(feature = "runtime-benchmarks")]
+impl BenchmarkHelper<MockAccountId, MockSeasonId, AssetId, Asset, ExampleTransitionId>
+	for SageBenchmarkHelper
+{
+	fn create_asset_for(account: &MockAccountId, season_id: &MockSeasonId, seed: u32) -> AssetId {
+		let asset_id = AssetId::from_low_u64_le(seed as u64);
+		let asset = Asset::create(asset_id, 0, 0, 0, [seed as u8; 32], 1, Level::One);
+
+		MockSeasonManager::register_asset_in(&asset_id, season_id)
+			.expect("Asset should be registered");
+		Assets::<Test, ()>::insert(asset_id, (account, asset));
+		AssetOwners::<Test, ()>::insert((account, season_id, &asset_id), ());
+
+		asset_id
+	}
+
+	fn create_bench_transition_for(
+		account: &MockAccountId,
+		season: &MockSeasonId,
+		seed: u32,
+	) -> (ExampleTransitionId, Vec<AssetId>) {
+		let asset_id_1 = Self::create_asset_for(account, season, seed);
+		let asset_id_2 = Self::create_asset_for(account, season, seed * 2);
+		let asset_id_3 = Self::create_asset_for(account, season, seed * 3);
+		let asset_id_4 = Self::create_asset_for(account, season, seed * 4);
+		let asset_id_5 = Self::create_asset_for(account, season, seed * 5);
+
+		let asset_vec = vec![asset_id_1, asset_id_2, asset_id_3, asset_id_4, asset_id_5];
+
+		(ExampleTransitionId::BenchTransition, asset_vec)
+	}
+}
```

Comment:
I like this. Although, it might be misleading for the downstream implementors that we have a `BenchTransition` here. They will not have that, they simply must know the most expensive transition.

But this can be changed later.